### ISSUE TITLE
Make it easier to load SpatiaLite

### ISSUE THREAD

**Issue by simonw** [simonw/datasette#668]
```
$ datasette spatial.db 
Serve! files=('spatial.db',) (immutables=()) on port 8001
ERROR: conn=<sqlite3.Connection object at 0x11e388f10>, sql = 'PRAGMA table_info(SpatialIndex);', params = None: no such module: VirtualSpatialIndex
Usage: datasette serve [OPTIONS] [FILES]...

Error: It looks like you're trying to load a SpatiaLite database without first loading the SpatiaLite module.

Read more: https://datasette.readthedocs.io/en/latest/spatialite.html
```
This error message could sniff around in the common locations for the SpatiaLite module and output the CLI command you should use to enable it:
```
datasette spatial.db --load-extension=/usr/local/lib/mod_spatialite.dylib
```
Even better: if Datasette had a `--spatialite` option which automatically loads the extension from common locations, if it can find it.

**Comment by simonw** [comment]
I can imitate how `shapefile-to-SQLite` does it: https://github.com/simonw/shapefile-to-sqlite/blob/5e3fb00eb1eee6ae0103a9ce4167a5e5ec17cf9f/shapefile_to_sqlite/utils.py#L105

**Comment by simonw** [comment]
I fixed this in #1115 - you can run `--load-extension=spatialite` now and it will look for the extension in common places.
